src/Razor/src/Microsoft.VisualStudio.Editor.Razor/Documents/DefaultFileChangeTracker.cs

```diff
@@ -18,8 +18,6 @@ public DefaultFileChangeTracker(string filePath)
             }
 
             FilePath = filePath;
-
-            GC.KeepAlive(Changed);
```

Comment:
lol what were we thinking

Reply:
It was an attempt to suppress CS0067 (which has now failed the build). I'll put in the proper fix.